src/test/java/fm/last/moji/integration/AbstractMojiIT.java

```diff
@@ -133,4 +133,62 @@ String newKey() {
     return newKey(RandomStringUtils.randomAlphanumeric(16));
   }
 
+  private void initMogileTestData() throws IOException {
+    if(initMogileTestDataIsDone) {
```

Comment:
Agree static method is not a problem, but I found another possible problem during modification: I cannot run `initMogileTestData()` with @BerforeClass just once during integration test. Instead, it will run as many times as number of class which extends `AbstractMojiIT` (twice in our case). It may add additional several seconds for twice run, what do you think?